static/app/views/starfish/utils/useSpansQuery.tsx

```diff
@@ -28,13 +28,15 @@ export function useSpansQuery<T = any[]>({
   enabled,
   referrer = 'use-spans-query',
   cursor,
+  view = 'webservice',
```

Comment:
There's an enum for this! `StarfishType`

Reply:
Sick, didn't know that, thanks for pointing out, will update!

Reply:
Made the changes!